src/Notifier.ts

```diff
@@ -95,9 +95,6 @@ export const Notifier = {
         if (!plaf.supportsNotifications() || !plaf.maySendNotifications()) {
             return;
         }
-        if (global.document.hasFocus()) {
```

Comment:
is there a check to prevent notifications for the currently active room somewhere?

Reply:
I think that is what is happening here: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/Notifier.ts#L404-L410